### ISSUE TITLE
Some strange behaviour

### ISSUE THREAD

**Issue by fimmind** [liquidz/vim-iced-coc-source#1]
## How to reproduce
Minimal init.vim (I use Neovim with vim-plug):
```vim
call plug#begin(stdpath('data') . '/plugged')
  Plug 'guns/vim-sexp'
  Plug 'ctrlpvim/ctrlp.vim'
  Plug 'liquidz/vim-iced', {'for': 'clojure'}
  Plug 'neoclide/coc.nvim', {'branch': 'release'}
  Plug 'liquidz/vim-iced-coc-source', {'for': 'clojure'}
call plug#end()

" Just for mappings
let g:maplocalleader=' '
let g:mapleader=' '
let g:iced_enable_default_key_mappings = v:true
```

and coc-settings.json:
```json
{
    "coc.source.iced.enable": true
}
```

Create new project with leiningen and open core namespace, e.g.
```sh
$ lein new newclj
$ cd newclj
$ nvim ~/src/newclj/core.clj
```

Then run :IcedJackIn to connet to the repl. Finally, type (Don't copy-paste) this somewhere in the file:
```clojure
; ==================
```

And it turns into such a strange thing:
![10 04 20_11:34:41:308410520](https://user-images.githubusercontent.com/44144507/78976857-bccd5f80-7b1f-11ea-8f51-babcefcb4324.png)

Another way is to delete the last bracket of `defn` from and type something after it:
![10 04 20_11:40:27:934999472](https://user-images.githubusercontent.com/44144507/78977038-1afa4280-7b20-11ea-8f75-bc36438b5194.png)

Highlighting appears automaticly and actually is a visual selection, therfore if I'll leave insert mode and type `d` highlighted part will be deleted.

**Comment by liquidz** [comment]
@fimmind Thanks for your reporting!
I could reproduce the problem.

I'll try to find out why.

**Comment by liquidz** [comment]
@fimmind It was a problem on not vim-iced-coc-source but [vim-iced](https://github.com/liquidz/vim-iced) itself.
https://github.com/liquidz/vim-iced/issues/202

I fixed in `dev` branch on vim-iced, so could you try this branch?

**Comment by fimmind** [comment]
This fixed the issue. Thanks :)

**Comment by liquidz** [comment]
@fimmind Sorry for late reply.
Thanks for your confirmation.
I just released ver 1.4.0!  (https://github.com/liquidz/vim-iced/pull/204)
